packages/qdrant-loader/src/qdrant_loader/utils/logging.py

```
@@ -545,6 +547,12 @@ def _mask(value: str) -> str:
                 except Exception:
                     return "***REDACTED***"
 
+            def _sanitize_string(value: str) -> str:
+                try:
+                    return redact_sensitive_data(value, mask="***REDACTED***")
+                except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)